synthtool/languages/node.py

```diff
@@ -278,6 +291,18 @@ def _noop(library: Path) -> None:
     pass
 
 
+def walk_through_apiary(dir, glob_to_search_for):
```

Comment:
A couple comments might be worthwhile:

`# walk through apiary to determine the current list of APIs that have been generated`